tls/descriptions/descriptions.md

```diff
@@ -69,754 +70,777 @@ To create a description, set the `openshift.io/description` annotation to the ma
 
       **Description:** 
       
 
-4. ns/openshift-kube-apiserver secret/check-endpoints-client-cert-key
+16. ns/openshift-kube-controller-manager configmap/aggregator-client-ca
 
       **Description:** 
       
 
-5. ns/openshift-kube-apiserver secret/control-plane-node-admin-client-cert-key
+17. ns/openshift-kube-controller-manager configmap/client-ca
 
       **Description:** 
       
 
-6. ns/openshift-kube-apiserver secret/external-loadbalancer-serving-certkey
+18. ns/openshift-route-controller-manager configmap/client-ca
 
       **Description:** 
       
 
-7. ns/openshift-kube-apiserver secret/internal-loadbalancer-serving-certkey
 
-      **Description:** 
-      
 
-8. ns/openshift-kube-apiserver secret/kubelet-client
+### kube-controller-manager (10)
+#### Certificates (3)
+1. ns/openshift-kube-controller-manager secret/csr-signer
 
       **Description:** 
       
 
-9. ns/openshift-kube-apiserver secret/localhost-recovery-serving-certkey
+2. ns/openshift-kube-controller-manager-operator secret/csr-signer
 
       **Description:** 
       
 
-10. ns/openshift-kube-apiserver secret/localhost-serving-cert-certkey
+3. ns/openshift-kube-controller-manager-operator secret/csr-signer-signer
 
       **Description:** 
       
 
-11. ns/openshift-kube-apiserver secret/service-network-serving-certkey
+
+
+#### Certificate Authority Bundles (7)
+1. ns/openshift-config-managed configmap/csr-controller-ca
 
       **Description:** 
       
 
-12. ns/openshift-kube-apiserver-operator secret/aggregator-client-signer
+2. ns/openshift-config-managed configmap/kubelet-serving-ca
 
       **Description:** 
       
 
-13. ns/openshift-kube-apiserver-operator secret/kube-apiserver-to-kubelet-signer
+3. ns/openshift-kube-apiserver configmap/kubelet-serving-ca
 
       **Description:** 
       
 
-14. ns/openshift-kube-apiserver-operator secret/kube-control-plane-signer
+4. ns/openshift-kube-controller-manager configmap/serviceaccount-ca
 
       **Description:** 
       
 
-15. ns/openshift-kube-apiserver-operator secret/loadbalancer-serving-signer
+5. ns/openshift-kube-controller-manager-operator configmap/csr-controller-ca
 
       **Description:** 
       
 
-16. ns/openshift-kube-apiserver-operator secret/localhost-recovery-serving-signer
+6. ns/openshift-kube-controller-manager-operator configmap/csr-controller-signer-ca
 
       **Description:** 
       
 
-17. ns/openshift-kube-apiserver-operator secret/localhost-serving-signer
+7. ns/openshift-kube-controller-manager-operator configmap/csr-signer-ca
 
       **Description:** 
       
 
-18. ns/openshift-kube-apiserver-operator secret/node-system-admin-client
+
+
+### kube-scheduler (1)
+#### Certificate Authority Bundles (1)
+1. ns/openshift-kube-scheduler configmap/serviceaccount-ca
 
       **Description:** 
       
 
-19. ns/openshift-kube-apiserver-operator secret/node-system-admin-signer
 
-      **Description:** 
+
+## Items That DO Meet the Requirement (124)
+### etcd (36)
```

Comment:
@hasbro17 @soltysh These are some pretty weak descriptions.  

A good descriptions is in the style of API documentation and includes

1. Which connections a CA bundle can be used to verify.
2. What kind of certificates a signer will sign for.
3. Which names and IPs a serving certificate terminates.
4. Which subject (user and group) a client certificate is created for.
5. Which binary and flags is this certificate wired to.


[ref](https://github.com/openshift/origin/blob/master/tls/descriptions/descriptions.md#how-to-meet-the-requirement)

Reply:
Noted. I think we overlooked that the descriptions had these requirements when we recently updated our cert controller.
Created a TODO for us to bring the descriptions up to spec.
https://issues.redhat.com/browse/ETCD-574